gp-includes/formats/format_pomo.php

```diff
@@ -31,11 +31,37 @@ public function print_exported_file( $project, $locale, $translation_set, $entri
 		foreach( $entries as $entry ) {
 			$po->add_entry( $entry );
 		}
-		$po->set_header( 'Project-Id-Version', $project->name );
+
+		$current = $project;
+		$project_tree = array();
+		$project_tree[] = $current->name;
+
+		while ( $current->parent_project_id > 0 ) {
+			$current = GP::$project->get( $project->parent_project_id );
+			$project_tree[] = $current->name;
+		}
+
+		$project_tree = array_reverse( $project_tree );
+
+		$project_id_version = implode( ' - ', $project_tree );
+
+		/**
+		 * Filter the project name before export.
+		 *
+		 * @since 2.1.0
+		 *
+		 * @param string $project_id_version The default project name to use in the header and
+		 *                                   comments ( "Parent - Child - GrandChild - etc." by default).
+		 * @param array  $project_tree       An array of the parent/child project tree, ordered from Parent
+		 *                                   to child to grandchild to etc...
+		 */
+		$project_id_version = apply_filters( 'gp_pomo_export_project_name', $project_id_version, $project_tree );
+
+		$po->set_header( 'Project-Id-Version', $project_id_version );
 
 		// TODO: include parent project's names in the comment
-		$po->comments_before_headers .= "Translation of {$project->name} in {$locale->english_name}\n";
-		$po->comments_before_headers .= "This file is distributed under the same license as the {$project->name} package.\n";
+		$po->comments_before_headers .= "Translation of {$project_id_version} in {$locale->english_name}\n";
+		$po->comments_before_headers .= "This file is distributed under the same license as the {$project_id_version} package.\n";
```

Comment:
It makes sense if you consider that a given project may have gone through different licenses in the past, so adding the version for specificity make as much sense as the rest of the line.

It get's weirder when you consider something like translate.w.org, which creates language packs for distribution without any "parent" project per say.

At this point I'm leaning more towards just removing these comment lines completely.